shared/locales/permissions/permissions.en-US.properties

```diff
@@ -27,6 +27,8 @@ perm-browser=Inspect Web Pages
 perm-camera=Cameras
 perm-camera-selector-appRequest=Select a camera for {{app}}.
 perm-camera-selector-webRequest=Select a camera for {{site}}.
+perm-camera-appRequest={{app}} would like to use your camera.
```

Comment:
If I remember correctly, it's not need to have `perm-camera-app/webRequest`. Do you find a scenario that show such wording?